ParkenDD/Colors.swift

```diff
@@ -22,20 +22,34 @@ struct Colors {
 	*/
 	static func colorBasedOnPercentage(_ percentage: Double, emptyLots: Int) -> UIColor {
 
-		let hue = 1 - (percentage * 0.3 + 0.7) // I want to limit this to the colors between 0 and 0.3
+        let lotDensityColor: UIColor = {
+            switch percentage {
+            case 0.6...1:
+                return UIColor(rgba: "#006a39")
+            case 0.16..<0.6:
+                return UIColor(rgba: "#1daa8c")
+            case 0.01..<0.16:
+                return UIColor(rgba: "#7f0304")
+            default:
+                return UIColor(rgba: "#5c5c5c")
+            }
+        }()
 
 		let useGrayscale = UserDefaults.standard.bool(forKey: Defaults.grayscaleUI)
 		if useGrayscale {
 			if emptyLots <= 0 {
 				return UIColor(red: 0.8, green: 0.8, blue: 0.8, alpha: 1.0)
 			}
-			return UIColor(red: CGFloat(1 - (hue+0.2)), green: CGFloat(1 - (hue+0.2)), blue: CGFloat(1 - (hue+0.2)), alpha: 1.0)
+            return lotDensityColor
```

Comment:
Your indentation seems off, check your Xcode settings so that they match the project's.

Reply:
I agree that it looks odd in that snippet, but I double-checked at it's set to 4 spaces. That seems to be consistent with the project.  Please let me know if you think that's wrong.

Reply:
What happens if you select lines 38-43 then press `control+i` to reindent? Does Xcode show changes in your Git?

Reply:
Nothing.  It looks fine in my Xcode.
<img width="729" alt="Screen Shot 2020-10-28 at 9 29 48 PM" src="https://user-images.githubusercontent.com/54075840/97518289-ee473480-1964-11eb-936c-8a1ff2bf0d0b.png">

Reply:
It looks the same, but the blue line on the left means your indentation is different. Fix it; in your Xcode settings > Text Editing > Indentation